webcompat/templates/home-page/form.html

```diff
@@ -1,5 +1,5 @@
 <section
-  class="wc-UISection js-ReportForm wc-ReportForm {% if form.errors %}is-opened{% else %}is-closed{% endif %} grid"
+  class="js-ReportForm {% if form.errors %}is-opened{% else %}is-closed{% endif %} grid"
```

Comment:
I don't think we actually use this `.js-ReportForm` class anywhere.... but we're using the id for sure. Maybe we just file a cleanup bug for that, if you want to be conservative.

Reply:
I've removed the form from the list above, as this is a separate PR. 
#1891 / #1898

Added comment to remove the class. https://github.com/webcompat/webcompat.com/issues/1898#issuecomment-348918302